### PR TITLE
Fix problem with property showInsertIconInAddButton

### PR DESCRIPTION
**Pull request by rafalwojcik**
Fix problem with property showInsertIconInAddButton don't hide plus button in MultivaluedSection

Problem mentioned in  #1135

### CODE REVIEW

File: Source/Core/Core.swift [hunk 1]
```diff
@@ -912,7 +912,7 @@ extension FormViewController : UITableViewDelegate {
 			}
             return .none
         }
-        if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1 {
+        if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1 && section.showInsertIconInAddButton {
             return .insert
         }
```
Comment (mtnbarreto) [inline Source/Core/Core.swift]: I think the proper code is: 
```swift
 if section.multivaluedOptions.contains(.Insert) && indexPath.row == section.count - 1{ 
      return section.showInsertIconInAddButton ? .insert : .none
}
```

Comment (rafalwojcik) [reply Source/Core/Core.swift]: It will fall through so my code is also proper one. I don't see any advantages compared to your code.

Comment (mtnbarreto) [reply Source/Core/Core.swift]: For the last row if showInsertIconInAddButton is false and multivalued section allows Insertion and deletion your code display a delete icon when it should display nothing. 

Hope this clarify something.

Comment (rafalwojcik) [reply Source/Core/Core.swift]: Yes, now I understand. Let me change this.

**Comment by rafalwojcik** [comment]
@mtnbarreto Changed :)

**Pull request merged**